scripts/test.sh

```diff
@@ -41,10 +41,12 @@ start_ganache() {
   if [ "$SOLIDITY_COVERAGE" = true ]; then
     node_modules/.bin/testrpc-sc --gasLimit 0xfffffffffff --port "$ganache_port" "${accounts[@]}" > /dev/null &
   else
-    node_modules/.bin/ganache-cli --gasLimit 0xfffffffffff "${accounts[@]}" > /dev/null &
+    node_modules/.bin/ganache-cli --gasLimit 0xfffffffffff --port "$ganache_port" "${accounts[@]}" > /dev/null &
   fi
 
   ganache_pid=$!
+
+  sleep 1
```

Comment:
Maybe a more straight forward solution would be to "really" wait for ganache like:
``` bash
#!/bin/bash

echo "Waiting for Ganache to launch on 8545..."

while ! nc -z localhost 8545; do
  sleep 0.1 # wait for 1/10 of the second before check again
done

echo "Ganache launched!"
```

Reply:
For sure that'd be more robust than my extremely naive approach, but to be quite honest I just wanted to get a quick fix out, since this is only rarely an issue anyway. I'd be glad to add this though if you're willing to open a PR! :)

Reply:
I will. Tomorrow... (late enough here in Germany)